frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx

```diff
@@ -434,7 +431,31 @@ export const GovInfoFormContainer: React.FunctionComponent = () => {
       return;
     }
     await handleUpdateUser(meId, data)
-      .then(() => {
+      .then((result) => {
+        if (result.isProfileComplete) {
+          if (result.isProfileComplete) {
+            sessionStorage.setItem("currentProfileStatus", "Complete");
```

Comment:
I suggest using the session storage functions from frontend/common/src/helpers/storageUtils.ts instead of sessionStorage directly.  Just in case we every decide to do something else with session storage.

Reply:
It is inside the handler function. Do you think its possible to call a function here?